src/script/lua_api/l_mapgen.cpp

```diff
@@ -991,11 +991,13 @@ bool read_deco_simple(lua_State *L, DecoSimple *deco)
 		return false;
 	}
 
-	if ((deco->deco_param2 < 0) || (deco->deco_param2 > 255)) {
+	param2 = getintfield_default(L, index, "param2", 0);
```

Comment:
That's a style comment.

As far as I can see, we mix declarations on top and c99-style declarations inline across the code, and neither is preferred nor commented on. This particular code section is already a mix.

Not worth the time imho :)